`av==10.0.0` breaks CI

The recent release of `av==10.0.0` breaks at least two tests: https://app.circleci.com/pipelines/github/pytorch/vision/21245/workflows/ecbc81ff-f864-4227-a35d-b4ed5e9f57ee/jobs/1724114?invite=true#step-108-3075

```
=================================== FAILURES ===================================
____________________ TestVideo.test_video_clips_custom_fps _____________________
Traceback (most recent call last):
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1120, in _try_get_data
    data = self._data_queue.get(timeout=timeout)
  File "/root/project/env/lib/python3.7/multiprocessing/queues.py", line 104, in get
    if not self._poll(timeout):
  File "/root/project/env/lib/python3.7/multiprocessing/connection.py", line 257, in poll
    return self._poll(timeout)
  File "/root/project/env/lib/python3.7/multiprocessing/connection.py", line 414, in _poll
    r = wait([self], timeout)
  File "/root/project/env/lib/python3.7/multiprocessing/connection.py", line 921, in wait
    ready = selector.select(timeout)
  File "/root/project/env/lib/python3.7/selectors.py", line 415, in select
    fd_event_list = self._selector.poll(timeout)
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/_utils/signal_handling.py", line 66, in handler
    _error_if_any_worker_fails()
RuntimeError: DataLoader worker (pid 5231) is killed by signal: Segmentation fault. 

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/project/test/test_datasets_video_utils.py", line 63, in test_video_clips_custom_fps
    video_clips = VideoClips(video_list, num_frames, num_frames, fps, num_workers=2)
  File "/root/project/torchvision/datasets/video_utils.py", line 132, in __init__
    self._compute_frame_pts()
  File "/root/project/torchvision/datasets/video_utils.py", line 153, in _compute_frame_pts
    for batch in dl:
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 628, in __next__
    data = self._next_data()
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1316, in _next_data
    idx, data = self._get_data()
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1282, in _get_data
    success, data = self._try_get_data()
  File "/root/project/env/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1133, in _try_get_data
    raise RuntimeError('DataLoader worker (pid(s) {}) exited unexpectedly'.format(pids_str)) from e
RuntimeError: DataLoader worker (pid(s) 5231) exited unexpectedly
----------------------------- Captured stderr call -----------------------------
100.0%
ERROR: Unexpected segmentation fault encountered in worker.
ERROR: Unexpected segmentation fault encountered in worker.

```

I'm going to pin to `< 10`. 

cc @seemethere

--- a/.circleci/unittest/linux/scripts/environment.yml
+++ b/.circleci/unittest/linux/scripts/environment.yml
@@ -13,4 +13,4 @@ dependencies:
   - pip:
     - future
     - scipy
-    - av
+    - av < 10

--- a/.circleci/unittest/windows/scripts/environment.yml
+++ b/.circleci/unittest/windows/scripts/environment.yml
@@ -14,6 +14,6 @@ dependencies:
   - pip:
     - future
     - scipy
-    - av != 9.1.1
+    - av !=9.1.1, <10
     - dataclasses
     - h5py

--- a/.github/workflows/test-linux-cpu.yml
+++ b/.github/workflows/test-linux-cpu.yml
@@ -56,7 +56,7 @@ jobs:
             -c nvidia "pytorch-${CHANNEL}"::pytorch[build="*${VERSION}*"] \
             "${CUDATOOLKIT}"
           ${CONDA_RUN} python3 setup.py develop
-          ${CONDA_RUN} python3 -m pip install pytest pytest-mock av
+          ${CONDA_RUN} python3 -m pip install pytest pytest-mock 'av<10'
       - name: Run tests
         shell: bash -l {0}
         env:

--- a/.github/workflows/test-m1.yml
+++ b/.github/workflows/test-m1.yml
@@ -37,7 +37,7 @@ jobs:
           conda create -yp ${ENV_NAME} python=${PY_VERS} numpy libpng jpeg scipy
           conda run -p ${ENV_NAME} python3 -mpip install --pre torch --extra-index-url=https://download.pytorch.org/whl/${CHANNEL}
           conda run -p ${ENV_NAME} python3 setup.py develop
-          conda run -p ${ENV_NAME} python3 -mpip install pytest pytest-mock av
+          conda run -p ${ENV_NAME} python3 -mpip install pytest pytest-mock 'av<10'
       - name: Run tests
         shell: arch -arch arm64 bash {0}
         env:

